### ISSUE TITLE
Scope misapplied with repeated subfields

### ISSUE THREAD

**Issue by ThomasDukleth** [lcnetdev/marc2bibframe#10]
When one MARC 21 subfield applies to multiple repeated subfields in a field collectively, the Bibframe conversion is out of scope.

**1.  Examples**

There may be many examples but the occurrence of multiple publishers for an instance seems the most readily accessible to me.

**1.1.  Publication Statement**

**1.1.1.  Bug**

**1.1.1.1.  MARC 21 LC Record Source**

[Some spaces added for better readability in this report.]

```
001 __ 468476
260 __ $a Geneva, Switzerland : $b Skira ; $a New York : $b Rizzoli, $c 1983.
```

**1.1.1.2.  BIBFRAME Conversion**

```
<bf:publication>
  <bf:ProviderEntity>
    <bf:providerName>Skira</bf:providerName>
    <bf:providerPlace>Geneva, Switzerland</bf:providerPlace>
    <bf:providerDate>1983.</bf:providerDate>
  </bf:ProviderEntity>
</bf:publication>
<bf:publication>
  <bf:ProviderEntity>
    <bf:providerName>Rizzoli</bf:providerName>
    <bf:providerPlace>New York</bf:providerPlace>
  </bf:ProviderEntity>
</bf:publication>
```

The date contained in the bf:providerDate property should apply to both providerEntity properties.  The date should be an actual date without trailing punctuation.

**1.1.2.  Possibly Mistaken Remedies**

Simply adding the common bf:providerDate property to the multiple bf:providerEntity properties may be unnecessarily redundant and a mistake for the case of transcribed publication information where the date is given only once in the material from which the information is transcribed.

```
<bf:publication>
  <bf:ProviderEntity>
    <bf:providerName>Skira</bf:providerName>
    <bf:providerPlace>Geneva, Switzerland</bf:providerPlace>
    <bf:providerDate>1983</bf:providerDate>
  </bf:ProviderEntity>
</bf:publication>
<bf:publication>
  <bf:ProviderEntity>
    <bf:providerName>Rizzoli</bf:providerName>
    <bf:providerPlace>New York</bf:providerPlace>
    <bf:providerDate>1983</bf:providerDate>
  </bf:ProviderEntity>
</bf:publication>
```

**1.1.3.  Remedies with Better Scope**

Add the property bf:publicationDate to the bf:publication property where a single date of publication applies to all bf:ProviderEntity properties.

A single bf:publication property should be able to include all subsidiary properties.  There should be no need to repeat the bf:publication property for each bf:ProviderEntity.

```
<bf:publication>
  <bf:ProviderEntity>
    <bf:providerName>Skira</bf:providerName>
    <bf:providerPlace>Geneva, Switzerland</bf:providerPlace>
  </bf:ProviderEntity>
  <bf:ProviderEntity>
    <bf:providerName>Rizzoli</bf:providerName>
    <bf:providerPlace>New York</bf:providerPlace>
  </bf:ProviderEntity>
  <bf:publicationDate>1983</bf:publicationDate>
</bf:publication>
```
